src/TLuaInterpreter.cpp

```diff
@@ -13128,9 +13157,14 @@ void TLuaInterpreter::initLuaGlobals()
     QString n;
     int error;
 
+#ifdef Q_OS_LINUX
     // if using LuaJIT, adjust the cpath to look in /usr/lib as well - it doesn't by default
     luaL_dostring (pGlobalLua, "if jit then package.cpath = package.cpath .. ';/usr/lib/lua/5.1/?.so;/usr/lib/x86_64-linux-gnu/lua/5.1/?.so' end");
```

Comment:
I think we have dropped support for LuaJIT in the development branch - this line was removed some time back from that branch IIRC...!